torch/_dynamo/variables/misc.py

```diff
@@ -722,7 +722,7 @@ def call_function(
                 path = inspect.getfile(self.value)
             except TypeError:
                 path = f"Builtin {self.value.__name__}"
-            unimplemented("call_function in skip_files " + path)
+            unimplemented(f"call_function {self.value} in skip_files {path}")
```

Comment:
The default printing for typical functions isn't great. This would be more useful as a qualname, if you can get it, or something.

Reply:
Yeah sg